src/MeoAssistant/Task/Miscellaneous/CreditFightTask.cpp

```diff
@@ -35,7 +35,7 @@ asst::CreditFightTask::CreditFightTask(const AsstCallback& callback, Assistant*
     stage_navigation_task_ptr->set_stage_name(copilot_task_ptr->get_stage_name());
 
     // 战斗结束后
-    stop_task_ptr->set_tasks({ "EndOfActionThenStop" });
+    stop_task_ptr->set_tasks({ "CreditFightEndOfAction" });
```

Comment:
```suggestion
    stop_task_ptr->set_tasks({ "EndOfActionThenStop", "FightMissionFailedThenStop" });
```

这样就不用 `CreditFightEndOfAction` 这个了